app/lib/features/chat_ng/widgets/events/chat_event_item.dart

```diff
@@ -35,11 +37,56 @@ class ChatEventItem extends StatelessWidget {
 
     return switch (eventType) {
       // handle message inner types separately
-      'm.room.message' => buildMsgEventItem(
-          context,
-          roomId,
-          messageId,
-          item,
+      'm.room.message' => Column(
```

Comment:
I think this is too much of a tree inside a `switch`, probably better to keep it in a function or better even as a separate widget.